guide/src/tutorials/draw/drawing-images.md

```diff
@@ -0,0 +1,188 @@
+# Drawing Images
+
+**Tutorial Info**
+
+- Author: Josiah Savary
+- Required Knowledge:
+    - [Getting Started](/getting_started.md)
+    - [Draw a Sketch](./draw-a-sketch.md)
+- Reading Time: 20 minutes
+
+---
+
+In this tutorial, we explore drawing images with nannou. We will cover loading textures from images the app's assets directory and drawing them, and applying basic transformations.
```

Comment:
Given the introductory nature of the tutorial, a sentence or two introducing what textures are could be useful  (versus drawing, etc)... or even just a link for additional reading.

Alternatively, this suggestion could make sense at the bottom of the document to provide a clean finish to the tutorial.

Reply:
@sbauer322 Okay I shuffled things around a bit and added a [What is a Texure?](https://github.com/jozanza/nannou/blob/drawing-images-tutorial/guide/src/tutorials/draw/drawing-images.md#what-is-a-texture) section. Let me know if that does the trick!